plugins/org.yakindu.sct.generator.c/library/FeatureTypeLibrary.xmi

```diff
@@ -11,13 +11,6 @@
     <parameters
         name="moduleName"
         optional="true"/>
-    <parameters
```

Comment:
Don't remove this one! 
* It makes sense for generic domain and it is likely that it is in use.
* it is fine that this setting has no effect if ShortIdentifiers annotation is used in C statecharts.
  * a warning in sgen files would be nice but is not mandatory.